mesonbuild/interpreter.py

```diff
@@ -2190,9 +2197,8 @@ def func_vcs_tag(self, node, args, kwargs):
             else:
                 vcs_cmd = [' '] # executing this cmd will fail in vcstagger.py and force to use the fallback string
         # vcstagger.py parameters: infile, outfile, fallback, source_dir, replace_string, regex_selector, command...
-        kwargs['command'] = [sys.executable,
-                             self.environment.get_build_command(),
-                             '--internal',
+        kwargs['command'] = self.environment.get_build_command() + \
+                            ['--internal',
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->